Validation/Geometry/src/MaterialBudgetHcalHistos.cc

```diff
@@ -41,11 +42,12 @@ void MaterialBudgetHcalHistos::fillBeginJob(const DDCompactView& cpv) {
     DDSpecificsMatchesValueFilter filter1{DDValue(attribute, value, 0)};
     DDFilteredView fv1(cpv, filter1);
     sensitives_ = getNames(fv1);
-    edm::LogVerbatim("MaterialBudget") << "MaterialBudgetHcalHistos: Names to be tested for " << attribute << " = "
-                                       << value << " has " << sensitives_.size() << " elements";
-    for (unsigned int i = 0; i < sensitives_.size(); i++)
-      edm::LogVerbatim("MaterialBudget") << "MaterialBudgetHcalHistos: sensitives[" << i << "] = " << sensitives_[i];
-
+    if (verbose_) {
```

Comment:
Hello @bsunanda, is this `verbose_` needed? Is it not possible to use `LogDebug`,`edm::LogInfo`, `edm::LogWarning` or `edm::LogError` and then manage the verbosity with the message logger?

Reply:
elm::LogVerbatim is used which is similar to elm::LogInfo without some detail. Using LogDebug requires recompiling the code to have the debug information which I want to avoid

Reply:
`LogInfo` (and thus `LogVerbatim`) messages are suppressed by default. What is the benefit of a `verbose_` configuration flag?

Reply:
When I activate the printouts for eta's in a certain range I may not need some of the printouts

Reply:
MessageLogger allows enabling/suppressing message at the granularity of modules and message categories. To me it looks like the module-level granularity would be functionally equivalent to a configuration flag.

Reply:
You are right. I can make use of that by changing the code